### ISSUE TITLE
Verification

### ISSUE THREAD

**Issue by jaambee** [shodgson/uareu#1]
Hi, how can I verify or enroll a fingerprint, to check after with a database?

Thanks !

**Comment by shodgson** [comment]
Hi Aldo,

For my purposes, I sent the image to a server for all processing and
authentication. For example, you can use an open source tool called
SourceAFIS.

**Comment by gbelmm** [comment]
![captura de pantalla 2018-02-08 a la s 12 19 51](https://user-images.githubusercontent.com/731936/35980800-686a3bc2-0cca-11e8-9470-28eeeb401052.png)


I have this error when trying to add SourceAFIS to the project

you could add it to android?

**Comment by shodgson** [comment]
No, I sent the image to a server where SourceAFIS processes the image.

**Comment by gbelmm** [comment]
you have another to do the same .. this does not work without having the latest version of java / android??

**Comment by shodgson** [comment]
Sorry, I don’t understand your question.

This code is only for capturing an image, and works on most android phones
that have OTG USB available.

**Comment by gbelmm** [comment]
an alternative to SourceAFIS?

**Comment by gbelmm** [comment]
fix me

in the project gradle.properties, add:
android.enableD8=true

**Comment by akbartriasno** [comment]
Dear gbelmm,

Can you share the code you are working on, i'm stuck to add sourceafis on android

Thanks before, sorry for my english

**Comment by saini391** [comment]
Dear gbelmm,
Can you please share the code thats working on your machine

**Comment by Ramzan55** [comment]
is there a way to verify/match fingerprints

**Comment by rogercarmona** [comment]
Hi, Shodgson.
I use your example of UareU in android (from https://github.com/shodgson/uareu) and I modified this for use the java libraries (from https://github.com/shodgson/uareulibrary) , and this work perfectly!!

You can help me to develop the library to obtain the enrroll template (for store in a database) and the library to verify the themplate with a finger.??? I can pay you!! 

Please write me by email to racarmon50@gmail.com. Thanks a lot

**Comment by shodgson** [comment]
Try looking at SourceAFIS. I think some people have used it natively on
Android, but I used it in a server.

Sorry I can’t help any further - life is a little too busy!

**Comment by rogercarmona** [comment]
Hi,

Please!!, help me to develop the library to obtain the enrroll template.
and I does the proofs in android.

I pay you!. please!!
